README.md

```diff
@@ -59,6 +59,7 @@ Options:
   -l                          List sections
   -d,--dom,--domain DOMAIN:{cfg,linux,stats,zoneCrab}
                               Abstract domain
+  --termination               Verify termination
```

Comment:
What's the use case for verifying programs that _don't_ terminate?  I.e., why not on by default?

Reply:
It should probably be on by default, I was thinking of the support as experimental. But also. see the other comment: maybe Prevail need not care at all, and just output a bound if known.